test/AutoDiff/SILOptimizer/derivative_sil.swift

```diff
@@ -69,6 +69,7 @@ func foo(_ x: Float) -> Float {
 // CHECK-SIL-LABEL: sil private [ossa] @fooTJpSpSr : $@convention(thin) (Float, @owned _AD__foo_bb0__PB__src_0_wrt_0) -> Float {
 // CHECK-SIL: bb0([[DY:%.*]] : $Float, [[PB_STRUCT:%.*]] : @owned $_AD__foo_bb0__PB__src_0_wrt_0):
 // CHECK-SIL:   [[ADD_PB:%.*]] = destructure_struct [[PB_STRUCT]] : $_AD__foo_bb0__PB__src_0_wrt_0
+// CHECK-SIL:   debug_value [[DY]] : $Float, let, name "y"
```

Comment:
You might want to compile with `-Xllvm -sil-print-debuginfo` and check that we also attach the correct scope, particularly if this was by cloning another function. You could check that the scope chain points back to this function and not to the original function that was cloned here (all assuming this code is result of a clone action of course :-).

Reply:
Similarly, this would allow you to check that the right location is attached, too.